pkg/logging/fluentd_logger.go

```diff
@@ -60,6 +74,18 @@ const (
 	maxReconnectInterval = 10 * time.Second
 )
 
+func FluentdLogOptsValidate(logOptMap map[string]string) error {
```

Comment:
`func (f *FluentdLogger) Validate`?

Reply:
I don't think making this function to be a struct method is a good idea. that means that the run.go need to init the struct , I don't think this would be necessary